client/state/jetpack-sync/actions.js

```diff
@@ -53,9 +52,8 @@ export function scheduleJetpackFullysync( siteId ) {
 			siteId,
 		} );
 
-		return wpcom
-			.undocumented()
-			.scheduleJetpackFullysync( siteId )
+		return wpcom.req
+			.post( { path: `/sites/${ siteId }/sync` }, {} )
```

Comment:
Can be just `.post( '/sites/...' )`. The empty `{}` param is not needed. The `useRemoveFollowerMutation` hook already does that kind of POST and it's verified that it really works.